### ISSUE TITLE
[Modeling] Scaffold DistilBERT fine-tune script in src/train.py

### ISSUE THREAD

**Issue by riancrtz** [riancrtz/fake-news-detection-nlp#6]
Set up the DistilBERT fine-tuning script using HuggingFace Transformers with early stopping on the validation set.
